ui/src/store/resource.test.ts

```diff
@@ -262,4 +264,201 @@ describe('Store functions', () => {
       }
     );
   });
+  it('it should return webURL', (done) => {
+    const store = ResourceStore.create(
+      {},
+      {
+        api,
+        categories: CategoryStore.create({}, { api })
+      }
+    );
+    expect(store.isLoading).toBe(true);
+    when(
+      () => !store.isLoading,
+      () => {
+        expect(store.isLoading).toBe(false);
+        expect(store.resources.size).toBe(7);
+
+        const webURL = store.resources.get('aws-cli')?.webURL;
```

Comment:
Can we use the assert function from the utils directory so that we handle the case of undefined